### PR TITLE
set `LP_REWARD` on poolpair

### LINKED ISSUE #133: set `LP_Reward` on poolpair to playground
#### What would you like to be added:
Adding incentive (reward) to poolpair

#### Why is this needed:
To allow some dev env (eg: mobile) to test

### PR DESCRIPTION
**Pull request by canonbrother**
#### What kind of PR is this?:

/kind feature

#### What this PR does / why we need it:
added **LP_REWARD** on poolpair

#### Which issue(s) does this PR fixes?:

Fixes #133

### CODE REVIEW

File: src/module.playground/setup/setup.dex.ts [hunk 1]
```diff
@@ -115,7 +115,11 @@ export class SetupDex extends PlaygroundSetup<PoolPairSetup> {
 
   async has (each: PoolPairSetup): Promise<boolean> {
     try {
-      await this.client.poolpair.getPoolPair(each.symbol)
+      const poolPair = await this.client.poolpair.getPoolPair(each.symbol)
+      const poolPairId = parseInt(Object.keys(poolPair)[0])
+      // '1' fairly divided by current 5 poolpairs -> 0.2
+      await this.client.masternode.setGov({ LP_SPLITS: { [poolPairId]: 0.2 } })
+      await this.generate(1)
```
Comment (fuxingloh) [inline src/module.playground/setup/setup.dex.ts:122]: logic should be at `create`

**Review by fuxingloh — CHANGES_REQUESTED** [review]

**Review by monstrobishi — APPROVED** [review]

**Comment by monstrobishi** [comment]
would be good to add a test to confirm that it comes through in `rewardPct`

File: src/module.playground/setup/setup.dex.ts (hunk 1 above)
Comment (canonbrother) [reply src/module.playground/setup/setup.dex.ts:122]: just realize it's required another implemetation to allow reward is fairly divided to each poolpair.
```
1. setgov `tokenId: 0.2` on first poolpair -> err as reward must be 1
2. place setgov `tokenId: 1` on create() -> no err but only the last poolpair contains reward
```

Comment (fuxingloh) [reply src/module.playground/setup/setup.dex.ts:122]: We need another `setup.rewards.ts`?

**Comment by canonbrother** [comment]
WIP

File: src/module.playground/setup/setup.dex.ts (hunk 1 above)
Comment (canonbrother) [reply src/module.playground/setup/setup.dex.ts]: nope.. need make use of the `after()`
i did some refactoring

File: src/module.playground/setup/setup.dex.ts [hunk 2]
```diff
@@ -121,4 +121,20 @@ export class SetupDex extends PlaygroundSetup<PoolPairSetup> {
       return false
     }
   }
+
+  async before (list: PoolPairSetup[]): Promise<void> {}
+
+  async after (list: PoolPairSetup[]): Promise<void> {
```
Comment (fuxingloh) [inline src/module.playground/setup/setup.dex.ts:127]:
```suggestion
  protected async after (list: PoolPairSetup[]): Promise<void> {
```

They are meant to be overridable but optional, hence you don't need to declare if you don't need them.

**Review by fuxingloh — CHANGES_REQUESTED** [review]

**Review by fuxingloh — APPROVED** [review]

**Pull request merged**